docs/en/setup/backend/static-gateways.md

```diff
@@ -0,0 +1,22 @@
+# Static Gateways/Proxies
```

Comment:
I prefer to use `Uninstrumented` to replace static, because gateway is always static :)